parfive/downloader.py

```diff
@@ -364,6 +372,11 @@ def callback(token, future, main_pb):
         """
         timeout = aiohttp.ClientTimeout(**timeouts)
         try:
+            if 'PROXY' in os.environ:
+                kwargs['proxy'] = os.environ['PROXY']
+            if 'PROXY_AUTH' in os.environ:
+                kwargs['proxy_auth'] = os.environ['PROXY_AUTH']
+
```

Comment:
So I have a question, given a session provided as a parameter, would it be safe to assume that the header information would not have to be explicitly passed to it?

Reply:
We allow passing the session in?

Reply:
As a `_get_http` parameter

Reply:
get_http is internal, so I am not sure I follow. Also as the `header=` argument is to the `get()` method and not the `ClientSession` then I think it's not relevant?